src/commands/setup.rs

```diff
@@ -118,10 +119,15 @@ impl Setup {
                         continue;
                     }
                     let id_network_hash = CoreUtils::create_network_hash(net_name, MAX_HASH_SIZE);
+                    let hash_int = status_block.to_owned().interfaces.unwrap();
```

Comment:
never use unwrap(), return a reasonable error instead. Also you should not get the option from the status. just parse the option from the network config here!